pyphare/pyphare/__main__.py

```
@@ -0,0 +1,264 @@
+    print_sim_summary(sim)
+    live = Simulator(sim, print_one_line=True)
+    live.initialize()
+    live.post_advance = PostOp(cli_args, live, update)
+
+    if cli_args.tui and cpp.mpi_rank() == 0:
+        from .tui import TextualTUI
+
+        tui = TextualTUI()
+        simulator.print = tui.print
+
+        t = Thread(target=live.run, daemon=True, args=[cli_args.monitor])
```

Ruff (0.15.10)

[error] Undefined name `Thread`

(F821)